src/Domain/CoordinateMaps/Wedge3D.cpp

```diff
@@ -143,27 +117,23 @@ Wedge3D::jacobian(const std::array<T, 3>& x) const noexcept {
   const ReturnType& xi = x[0];
   const ReturnType& eta = x[1];
   const ReturnType& zeta = x[2];
-  const double ratio_of_radii =
-      radius_of_spherical_surface_ / radius_of_other_surface_;
-  const ReturnType x_equiangular_secant =
-      make_with_value<ReturnType>(xi, 1.0) / cos(M_PI_4 * xi);
-  const ReturnType y_equiangular_secant =
-      make_with_value<ReturnType>(xi, 1.0) / cos(M_PI_4 * eta);
-  const ReturnType x_equiangular_tangent = tan(M_PI_4 * xi);
-  const ReturnType y_equiangular_tangent = tan(M_PI_4 * eta);
-
-  const ReturnType sy_square = square(y_equiangular_secant);
-  const ReturnType sx_square = square(x_equiangular_secant);
-  const ReturnType scaling_and_sphericity = M_PI *
-      (ratio_of_radii * (1 + zeta) + sphericity_of_other_surface_ * (1 - zeta));
-  const ReturnType denominator_dzeta =
-      (2.0 * sqrt(sx_square + square(y_equiangular_tangent)));
-  const ReturnType denominator_other = pow<3>(denominator_dzeta);
-  const ReturnType nonsphericity = (-1.0 + sphericity_of_other_surface_) * 0.5 *
-                                   denominator_dzeta / sqrt(3.);
-  const ReturnType trigonometric_nonsphericity =
-      nonsphericity * (-1.0 + zeta) *
-      (3 + cos(M_PI_2 * xi) + 2 * cos(M_PI_2 * eta) * square(sin(M_PI_4 * xi)));
+
+  const double r_other = radius_of_other_surface_;
+  const double r_sphere = radius_of_spherical_surface_;
+  const double sphericity = sphericity_of_other_surface_;
```

Comment:
Why bother with the three copies above?

Reply:
The shorter variable names are so the jacobian is easier to read